gcp_variant_transforms/vcf_to_bq_common_test.py

```diff
@@ -24,7 +24,7 @@
 from gcp_variant_transforms.vcf_to_bq_common import PipelineModes
 
 
-class DataProcessorTest(unittest.TestCase):
+class VcfToBqCommonTest(unittest.TestCase):
   """Tests cases for the ``general_process`` script."""
```

Comment:
Can you also help update the doc string here? s/general_process/vcf_to_bq_common.

Reply:
Done! Thanks for catching this one too!